index.ts

```diff
@@ -613,6 +616,138 @@ export const deserializeIlpRejection = (binary: Buffer): IlpRejection => {
   }
 }
 
+export interface IlpPrepare {
+  amount: string,
+  executionCondition: Buffer,
```

Comment:
I think we should have the expires at field before the condition so the two fields that change come first

Reply:
I originally advocated for that, but executionCondition and amount are fixed length, and expiresAt might not be